docs/update_bios.rst

```diff
@@ -0,0 +1,52 @@
+BIOS Updates
+============
+
+Below are the steps for updating the BIOS on the app and mon servers. You can also follow these steps to update the BIOS on your computer, such as the Admin Workstation Computer.
+
+What you need
+~~~~~~~~~~~~~
+
+  #. A clean USB device to download the ``.bio`` file
+  #. An internet-connected workstation, such as the *Admin Workstation*
+  #. A UPS (uninterrupted power supply), such as a surge-protecting power supply with a backup battery
+  #. A keyboard and mouse to plug into the servers (you can skip this if you are updating a laptop.)
```

Comment:
A mouse is not required for the NUC7 BIOS update process.

Reply:
yes - this was a brain fart (should have said monitor) and has been corrected!